com.amazon.Workspaces.yml

```diff
@@ -163,12 +184,6 @@ modules:
       - type: extra-data
         filename: workspacesclient.deb
         only-arches: [x86_64]
-        url: https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/focal/main/binary-amd64/workspacesclient_4.7.0.4312_amd64.deb
-        sha256: 1b4a39b859c49225169a44cc507955722770fb6c7c7bf2a631f54113ba0b5d5f
-        size: 46652448
-        x-checker-data:
-          type: debian-repo
-          package-name: workspacesclient
-          root: https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu
-          dist: focal
-          component: main
+        url: https://d3nt0h4h6pmmc4.cloudfront.net/new_workspacesclient_focal_amd64.deb
```

Comment:
we should copy this to archive.org or something so this can be reliably built since they for whatever reason do not have old builds or versioned files. Latest is 2024.0.4661.